odoo/fields.py

```diff
@@ -2487,6 +2489,11 @@ def _description_selection(self, env):
         else:
             return selection
 
+    def _default_group_expand(self, records, groups, domain, order):
+        # return a group per selection option, in definition order
+        selection = self.selection(records) if callable(self.selection) else self.selection
+        return [key for key, val in selection]
```

Comment:
`self.selection` can also be a method name.
This trivial implementation does the job :wink: 
```python
return self.get_values(records.env)
```